### PR TITLE
Replace all JSON Schema uses with Cleaners

### PR DESCRIPTION
**Pull request by swansontec**
I have managed to test everything except the EOS receive logic. The EOS node we use in `checkIncomingTransactions` simply isn't returning any results, even with incoming transactions, so the cleaner never runs.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203483455408818

### CODE REVIEW

File: src/ethereum/ethSchema.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const asEtherscanGetBlockHeight = asObject({
 })
 
 export const asEtherscanGetAccountNonce = asObject({
-  result: asString
+  result: asHexString
```
Comment (swansontec) [inline src/ethereum/ethSchema.ts:28]: I got this wrong originally, but testing quickly revealed the issue.

File: src/eos/eosSchema.ts [hunk 2]
```diff
@@ -121,31 +121,17 @@ query ($query: String!, $limit: Int64, $low: Int64, $high: Int64) {
...
-    required: ['act', 'trx_id', '@timestamp', 'block_num']
-  }
-}
+export const asEosTransactionSuperNodeSchema = asObject({
+  act: asObject({
+    data: asObject({
+      from: asString,
+      to: asString,
+      amount: asNumber,
+      symbol: asString,
```
Comment (peachbits) [inline src/eos/eosSchema.ts:130]: This appears to be optional

Comment (peachbits) [reply src/eos/eosSchema.ts:130]: Oof, nevermind it's required. The schema was wrong.

**Review by peachbits — APPROVED** [review]

**Pull request merged**